package/yast2-ntp-client.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri Mar  8 14:39:23 UTC 2019 - knut.anderssen@suse.com
+
+- Added cwm/popup dependency (bnc#893065, bsc#1039985)
```

Comment:
please use in both case same shortcut. or if you want emphasize suse internal as bsc and opensuse bug as boo I am fine. but bnc is now dead and should not be used anymore.